randovania/games/dread/gui/ui_files/games_tab_dread_widget.ui

```diff
@@ -183,80 +192,27 @@
        </property>
        <property name="textFormat">
-        <enum>Qt::PlainText</enum>
+        <enum>Qt::AutoText</enum>
```

Comment:
Could you try setting the text format to MarkdownText instead of changing it the pseudo-html format?